network_listview_builder/lib/main.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:network_listview_builder/reloadListView.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Network ListView Builder',
+      theme: ThemeData(
+        visualDensity: VisualDensity.adaptivePlatformDensity,
```

Comment:
`visualDensity` is undefined for me, had to remove it to run the app.

Remove the debug banner